### DISCUSSION TITLE
Dictionaries flagging correctly spelled words

### DISCUSSION THREAD

**Discussion by productiontriangle** [SubtitleEdit/subtitleedit#8563] [Q&A] [answered]
Anyone had this issue? Have not had a chance to test in other files so could be isolated. 

Random correctly spelled words are flagged as spelled incorrectly. Same across the 3 dictionaries I tested (US, GB, AU).  So I wonder if it is a formatting issue in the file (raw captions imported from Rev) rather than the dictionaries not working properly?

![image](https://github.com/SubtitleEdit/subtitleedit/assets/173519970/5115dc7f-467f-401a-8640-35d001581a57)

**Comment by niksedk** [comment]
Please attach the actual subtitle file. It's possible some letters are not actually what they look like, but some Unicode look-a-like letter...

**Comment by productiontriangle** [comment]
Sure, I have emailed the file to you. Thanks!

**Comment by niksedk** [comment, marked as answer]
Sure, it's weird Unicode code letter.s

"readу" uses Unicode Character “У” (U+0423) which is Cyrillic Capital Letter U
"Аnd " uses Unicode Character “А” (U+0410) which is Cyrillic Capital Letter A
etc...

**Reply by productiontriangle** [reply]
Thanks for identifying the problem!

*shakes fist at Rev*
